Content.Shared/GameObjects/Components/Chemistry/SolutionComponent.cs

```diff
@@ -52,6 +53,8 @@ public SolutionCaps Capabilities
             set => _capabilities = value;
         }
 
+        public ReadOnlyCollection<Solution.ReagentQuantity> ReagentList => _containedSolution.Contents;
```

Comment:
Following on from the above,
```suggestion
        public IReadOnlyList<Solution.ReagentQuantity> ReagentList => _containedSolution.Contents;
```